src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -213,6 +213,7 @@ type querySortType = "Ascending" | "Descending" | "Range Ascending" | "Range Des
 interface recentErrorDto extends Raven.Server.NotificationCenter.Notifications.Notification {
     Details: string;
     HttpStatus?: string;
+    LicenseLimitType?: string;
```

Comment:
use type from server? Optionally introduce new interface: recentLicenseErrorDto - not every error has field: LicenseLimitType

Reply:
done